client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -754,10 +799,23 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
     # go through AOVs in expected files
     for aov, files in expected_files[0].items():
         collected_files = _collect_expected_files_for_aov(files)
-
-        expected_filepath = collected_files
-        if isinstance(collected_files, (list, tuple)):
-            expected_filepath = collected_files[0]
+        first_filepath = collected_files
+        if isinstance(first_filepath, (list, tuple)):
+            first_filepath = first_filepath[0]
+        staging_dir = os.path.dirname(first_filepath)
+
+        if (
+            frames_to_render is not None
+            and isinstance(collected_files, (list, tuple))  # not single file
+        ):
+            frames_to_render = convert_frames_str_to_list(frames_to_render)
```

Comment:
This is wrong and bugged.
Client reported a bug, they got an error: `AttributeError: 'list' object has no attribute 'split'``

In this case it seems we're passing a `list` to `convert_frames_str_to_list` which does not handle `list` inputs